### PR TITLE
Explore Layers

### PR DESCRIPTION
**Pull request by sarveshpro**
Changes in this pull request:
- Explore Toggles

### CODE REVIEW

**Comment by sarveshpro** [comment]
Does anyone have an insight on how to use ESRI tiles in Mapbox?
https://storymaps.esri.com/stories/2019/anthropocene-atlas/3-forests.html
https://tiles.arcgis.com/tiles/nzS0F0zdNLvs7nc8/arcgis/rest/services/Tree_Cover_2000_Bright_Green/MapServer/4?f=pjson

File: src/features/public/Donations/components/MapboxMap.tsx [hunk 1]
```diff
@@ -371,9 +452,151 @@ export default function MapboxMap({
...
+            <Layer id="forest-layer" source="forests" type="raster" />
+          </Source>
+        ) : null}
+
+        {explorePotential ? (
+          <Source
+            id="potential"
+            type="raster"
+            tiles={[
+              ' https://earthengine.googleapis.com/map/80c988d5e8f6021ef9e6d2447f405c79/{z}/{x}/{y}?token=a5a5893006a0ea587845708b078df9ca',
```
Comment (norbertschuler) [inline src/features/public/Donations/components/MapboxMap.tsx:473]: Trying this locally is giving me errors right now:
```
Request URL: https://earthengine.googleapis.com/map/80c988d5e8f6021ef9e6d2447f405c79/3/1/1?token=a5a5893006a0ea587845708b078df9ca
Request Method: GET
Status Code: 404 
```

File: src/features/public/Donations/components/MapboxMap.tsx [hunk 2]
```diff
@@ -371,9 +452,151 @@ export default function MapboxMap({
...
+                : {}
+            }
+          >
+            {exploreExpanded ? <CancelIcon /> : <ExploreIcon />}
+            {exploreExpanded ? null : (
+              <p
+                onClick={() => setExploreExpanded(true)}
+                className={styles.exploreText}
+              >
+                {isMobile ? null : 'Explore'}
```
Comment (norbertschuler) [inline src/features/public/Donations/components/MapboxMap.tsx:524]: Please add `Explore` as a new translation resource with i18n here!

File: src/features/public/Donations/components/MapboxMap.tsx [hunk 3]
```diff
@@ -371,9 +452,151 @@ export default function MapboxMap({
...
+                  <div className={styles.exploreToggleRow}>
+                    <FormControlLabel
+                      control={
+                        <Switch
+                          checked={exploreForests}
+                          onChange={handleExploreForestsChange}
+                          name="forest"
+                        />
+                      }
+                      label="Forest"
```
Comment (norbertschuler) [inline src/features/public/Donations/components/MapboxMap.tsx:541]: Please also use translation resources for all of the labels.

**Review by norbertschuler — CHANGES_REQUESTED** [review]

**Pull request closed without merging**